src/main/resources/view/TaskCard.fxml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.layout.ColumnConstraints?>
+<?import javafx.scene.layout.GridPane?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.RowConstraints?>
+<?import javafx.scene.layout.VBox?>
+
+<HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+  <GridPane HBox.hgrow="ALWAYS">
+    <columnConstraints>
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+    </columnConstraints>
+    <VBox alignment="CENTER_LEFT" minHeight="70.0" GridPane.columnIndex="0">
+      <padding>
+        <Insets bottom="5" left="15" right="5" top="5" />
+      </padding>
+      <Label fx:id="label" styleClass="cell_small_label" text="\$label" />
+         <Label fx:id="date" styleClass="cell_small_label" text="\$date" />
+         <CheckBox fx:id="isDone" disable="true" mnemonicParsing="false" selected="true" text="Done" />
+    </VBox>
+      <rowConstraints>
+         <RowConstraints />
+      </rowConstraints>
```

Comment:
Any reason why there is an extra indentation here?

Reply:
No, it shouldn't be indented.